samples/frontend-backend/frontend/Program.cs

```diff
@@ -14,9 +14,6 @@ public static void Main(string[] args)
         {
             Activity.DefaultIdFormat = ActivityIdFormat.W3C;
             CreateHostBuilder(args).Build().Run();
-
```

Comment:
This is the best part of this change wtf.

Reply:
@JamesNK

Reply:
uhhh dotnet watch testing stuff.......

Reply:
Stop right there! You are under arrest by the whitespace police

![image](https://user-images.githubusercontent.com/303201/87611619-9a6d6280-c75c-11ea-81ab-f4e6aca6e5be.png)